apps/playground-sveltekit/src/lib/next-auth.ts

```diff
@@ -65,7 +65,7 @@ async function SKNextAuthHandler(
     query: Object.fromEntries(url.searchParams),
     headers: request.headers,
     method: request.method,
-    cookies: cookie.parse(request.headers.get("cookie")),
+    cookies: cookie.parse(request.headers.get("cookie") ?? ""),
```

Comment:
A nicer way to handle this would be:
```ts
  const headerCookie = request.headers.get("cookie")
  const cookies = headerCookie ? cookie.parse(headerCookie) : undefined
  // ...
  headers: request.headers,
  method: request.method,
  cookies,
```
In this way, there's no need to try to parse the cookies if there are not in the request.